test/MusicStore.E2ETests/nginx.conf

```diff
@@ -0,0 +1,37 @@
+﻿error_log [errorlog];
```

Comment:
Is it copied from somewhere? Can we have a default version in Hosting?

Reply:
Copied from server tests. Filed https://github.com/aspnet/Hosting/issues/1423